portability/toku_instr_mysql.h

```diff
@@ -12,8 +12,16 @@
 // undefine them here to avoid compilation errors.
 #undef __STDC_FORMAT_MACROS
 #undef __STDC_LIMIT_MACROS
-#include <mysql/psi/mysql_file.h>    // PSI_file
-#include <mysql/psi/mysql_thread.h>  // PSI_mutex
+#include "mysql/psi/mysql_file.h"    // PSI_file
+#include "mysql/psi/mysql_thread.h"  // PSI_mutex
+
+#if (MYSQL_VERSION_ID >= 80000)
+#include "mysql/psi/mysql_cond.h"
+#include "mysql/psi/mysql_memory.h"
+#include "mysql/psi/mysql_mutex.h"
```

Comment:
mysql_memory.h and mysql_stage.h are present in 5.7, thus their include should not depend on version >= 8.0 ?

Reply:
Perhaps.  5.6 does not have mysql_memory.h so there would need to be an additional conditional for >= 50700 and both 5.6 and 5.7 do not explicitly need these as they are pulled in through some other header.